packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -207,21 +220,38 @@ function* doStartPurchaseContentFlow({
         .div(BN_USDC_CENT_WEI)
         .toNumber()
       yield* put(buyUSDC())
-      yield* put(
-        onrampOpened({
-          provider: USDCOnRampProvider.STRIPE,
-          purchaseInfo: {
-            desiredAmount: balanceNeededCents
-          }
+      let result: { succeeded?: any; failed?: any; canceled?: any } | null =
```

Comment:
`Nullable<...>`? 😁 
Also maybe worth typing these if it's not too much of a hassle? It's going to be one of two actions for each state, right?